tame.lua

```diff
@@ -0,0 +1,49 @@
+-- tame and train animals {set animals as Wild through to Masterfully Trained}
+
+local help=[====[
+tame
+====
+Tame and train animals.
+
+Usage: `tame -read` OR `tame -set <level>`

```

Comment:
All of these backticks should be double backticks

Reply:
Done. It'll be in the next commit.